lib/models/DataPoint.dart

```diff
@@ -0,0 +1,33 @@
+class DataPoint {
+  DateTime date = DateTime.now();
+  int dayOfCycle = 0; // 1, 2, ...
+  Severity flow = Severity.unset; // LOW, MEDIUM, HIGH
+  int painLevel = 1; // 1-10
```

Comment:
0 through 10 might make more sense than 1 through 10, since "no pain" doesn't really correlate to 1